apps/moderation/src/components/transparency-log.tsx

```diff
@@ -119,7 +82,7 @@ const TransparencyLog: React.FC<ITransparencyLogProps> = props => {
     <Box>
       <SwitchCard
         count={
-          activeButton === 'All' ? count.kept + count.delisted : count[activeButton.toLowerCase()]
+          activeButton === 'All' ? count?.kept + count?.delisted : count[activeButton.toLowerCase()]
```

Comment:
here it would be a better approach to create an enum with the possible values of the buttons and use that instead of text values, ex.: enum ButtonValues { ALL: 'all', KEPT: 'kept'} and use ButtonValues.ALL instead of 'All' directly